examples/abitest/module/rust/src/lib.rs

```diff
@@ -173,22 +178,16 @@ fn invalid_raw_offset() -> *mut u64 {
 
 impl FrontendNode {
     fn test_channel_create_raw(&self) -> std::io::Result<()> {
-        let mut write = oak::WriteHandle { handle: 0 };
-        let mut read = oak::ReadHandle { handle: 0 };
+        let mut write = 0u64;
+        let mut read = 0u64;
         unsafe {
             expect_eq!(
                 OakStatus::ERR_INVALID_ARGS.value() as u32,
-                oak::wasm::channel_create(
-                    invalid_raw_offset() as *mut u64,
-                    &mut read.handle as *mut u64
-                )
+                oak::wasm::channel_create(invalid_raw_offset() as *mut u64, &mut read as *mut u64)
```

Comment:
(pre-existing) Are the `as *mut u64` casts needed?  (Passim, with some `0` -> `0u64` conversions possibly needed.)

Reply:
Some are needed for the ref to ptr part of the cast, but others can be removed. I think abstracting the following somewhere makes sense since this is used in a number of "raw" tests. 

~~~rust
fn channel_create_raw(&self) -> (u64,u64) {
    let mut w = 0u64;
    let mut r  = 0u64;
    unsafe { oak::wasm::channel_create(&mut w as *mut u64, &mut r as *mut u64) as i32; }
    (w,r)
}
~~~